src/client/pages/OfferNew/Checkout/UserDetailsForm.tsx

```diff
@@ -114,15 +124,9 @@ export const UserDetailsForm: React.FC<Props> = ({
           pattern="[0-9]*"
           maxLength={ssnMaxLength}
           value={ssn}
-          // errors={ssnError ? textKeys.SIGN_SSN_CHECK() : undefined} TODO error handling?
```

Comment:
lol, apparently _someone_ thought of this at some point in history 😬

![](https://i.imgur.com/FnrC70C.png)